src/components/Client/Register/index.jsx

```
@@ -1,109 +1,199 @@
+      await registerUser({
+        email: data.email,
+        phone_number: data.phoneNumber,
+        password: data.password,
+        name: data.userName
+      });
+      toast.success("Tạo tài khoản thành công, hãy đăng nhập!");
+      navigate("/login");
+    } catch (err) {
+      if (err.response?.data?.data?.errors) {
+        const { errors: validationErrors } = err.response?.data?.data;
+        if (validationErrors.email) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)